src/components/RadioButton/styles.ts

```diff
@@ -16,9 +20,9 @@ export const Radio = styled(Touchable)<RadioProps>`
   width: ${({ size }: RadioProps) => moderateScale(size)}px;
   height: ${({ size }: RadioProps) => moderateScale(size)}px;
   border-radius: ${({ size }: RadioProps) => moderateScale(size / 2)}px;
-  border: ${moderateScale(1)}px
+  border: ${smBorderWidth}px
     ${({ radioButtonColor }: RadioProps): any =>
-      hasColor(radioButtonColor, textColor)};
+      hasColor(radioButtonColor, textColor)}${hasColor('', '4D')};
```

Comment:
Não entendi essa parte :eyes:  `${hasColor('', '4D')};`

Reply:
No design nós temos que a cor da borda externa é a textColor com opacidade de 30%. 
Se eu colocar só um "opacity: 0.3" ele não funciona para a borda. Daí a alternativa foi colocar no hexadecimal o alfa da transparência (https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4). 
Assim, se quem for usar a lib passar um radioButtonColor, vamos aplicar essa cor diretamente na borda, porém se não for passado uma cor para essa borda (e consequentemente o componente for usar a textColor) nós adicionamos os 30% de opacidade no alfa (4D).

Reply:
Então basicamente:
- hasColor é true? Usamos a cor que o dev passou
- hasColor é false? Usamos textColor + 4D = #16254A4D

Reply:
Perfeito entendi!